Vagrantfile

```diff
@@ -159,7 +159,7 @@ Vagrant.configure(2) do |config|
       # Provider-independent options
       machine.vm.hostname = "gd2-#{node_index}"
       machine.vm.synced_folder '.', '/vagrant', disabled: true
-      machine.vm.synced_folder '/home/smukherjee/dev/redhat/tendrl/golang/src/github.com/gluster/glusterd2/build/', '/vagrant'
+      machine.vm.synced_folder '/home/dpivonka/golang/src/github.com/gluster/glusterd2/build/', '/vagrant'
```

Comment:
the path to find glusterd2 builds on vagrant vms, could you make this configurable?

@shirshendu comments?

Reply:
that shouldn't be hard. I can make it so its stored in the tendrl.conf.yml